aggregator/src/aggregator.rs

```diff
@@ -4662,6 +4666,45 @@ mod tests {
 
     #[tokio::test]
     async fn upload_filter() {
+        async fn check_response_fn(
+            response: &mut Response,
+            desired_status: StatusCode,
+            desired_type: &str,
+            desired_title: &str,
+            desired_task_id: &TaskId,
+            file: &str,
+            line: u32,
+        ) {
+            assert_eq!(response.status(), desired_status, "at {file}:{line}");
+            let problem_details: serde_json::Value =
+                serde_json::from_slice(&body::to_bytes(response.body_mut()).await.unwrap())
+                    .unwrap();
+            assert_eq!(
+                problem_details,
+                json!({
+                    "status": desired_status.as_u16(),
+                    "type": format!("urn:ietf:params:ppm:dap:error:{desired_type}"),
+                    "title": desired_title,
+                    "taskid": format!("{desired_task_id}"),
+                }),
+                "at {file}:{line}"
+            )
+        }
+
+        macro_rules! check_response {
```

Comment:
Rather than inventing a macro to use in one test, IMO we should just call `check_response_fn` directly, but remove the file/line arguments. Even without the file/line information included manually we can run with `RUST_BACKTRACE=1` to get accurate caller information when needed.

Reply:
We may want to revisit `profile.dev.debug = 0` in `Cargo.toml`, I've found myself commenting it out often. We could move it to a separate CI-only profile, perhaps.